packages/paste-website/src/pages/components/chat-composer/index.mdx

```diff
@@ -0,0 +1,225 @@
+        description
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Chat Composer"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/chat-composer"
+  storybookUrl="/?path=/story/components-chat-composer--default"
+  data={props.data}
+  description={props.data.mdx.frontmatter.description}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<LivePreview
+  noInline
+  language="jsx"
+  scope={{
+    Box,
+    ChatLog,
+    ChatBookend,
+    ChatBookendItem,
+    ChatEvent,
+    ChatMessage,
+    ChatBubble,
+    ChatMessageMeta,
+    Avatar,
+    ChatAttachment,
+    ChatAttachmentDescription,
+    ChatAttachmentLink,
+    ChatMessageMetaItem,
+    ChatComposer,
+    AutoScrollPlugin,
+    Button,
+    SendIcon,
+  }}
+>
+  {ChatDialogExample}
+</LivePreview>
+
+## Guidelines
+
+### About Chat Composer
+
+A Chat Composer is an input made for users to type rich chat messages. Chat Composer can be used as one part of larger chat user interfaces to provide a seamless authoring experience.
+
+### Accessibility
+
+Chat Composer supports a variety of aria attributes which are passed into the content editable region of the component.
+
+### Lexical and Plugins
+
+Chat Composer is built ontop of [Lexical](https://lexical.dev/). The editor itself is extensible and follows a declarative approach to configuration via JSX. Developers can leverage a
+wide variety of existing plugins or write their own to augment the behavior of Chat Composer. To include new Chat Composer behavior provide plugins via the `children` prop.
+
+## Examples
+
+### Basic
+
+<LivePreview scope={{ChatComposer}} language="jsx">
+  {`<ChatComposer config={{namespace: 'foo', onError: (e) => { throw e } }} ariaLabel="A Basic Chat Composer" />`}
+</LivePreview>
+
+### With Placeholder
+
+<LivePreview scope={{ChatComposer}} language="jsx">
+  {`<ChatComposer config={{namespace: 'foo', onError: (e) => { throw e } }} placeholder="Type Here..." ariaLabel="A Basic Chat Composer" />`}
+</LivePreview>
+
+### With Initial Value
+
+<LivePreview scope={{ChatComposer}} language="jsx">
+  {`<ChatComposer config={{namespace: 'foo', onError: (e) => { throw e } }} initialValue="This is my initial value" placeholder="Type Here..." ariaLabel="A Basic Chat Composer" />`}
+</LivePreview>
+
+### With Max Height
+
+<LivePreview scope={{ChatComposer}} language="jsx">
+  {`<ChatComposer config={{namespace: 'foo', onError: (e) => { throw e } }} maxHeight="size10" ariaLabel="A Basic Chat Composer" />`}
+</LivePreview>
+
```

Comment:
todo: let's include an example where we use a lexical plugin in composition with the component, like the scrolling example Lee showed us. It's such a cool thing that this component can do, melds right into the principles of our system. Let's show it off!!

Reply:
I think it'd be cool to show how one would extend this component if Lee already has an easy to port example built. If building from scratch, we can omit until it comes up. But I think this component/lexical by nature is something that is a composed UI for different products, so showcasing that here could be powerful.

Reply:
I think theres a markdown formatter you could use? That one is fun... or they have a TON of example plugins here: https://github.com/facebook/lexical/tree/main/packages/lexical-playground/src/plugins